### PR TITLE
(docs) update README.md

### PR DESCRIPTION
**Pull request by vladdoster**
- Update wording
- provide relevant links
- make the FAQ image smaller
- make disclaimer clearer
- wrap lines to make it easier to read from CLI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,64 +60,89 @@ Run the following line in the terminal: `pip install -r requirements.txt`.
...
-SCOUT_MULTIPLIER: 5
-SCOUT_SLEEP_TIME: 5
-TLD: com
+To set an environment variable, run:
+
+```bash
+export supported_coin_list="XLM TRX ICX EOS IOTA ONT QTUM ETC ADA XMR DASH NEO ATOM DOGE VET BAT OMG BTT"
+
+# numbers don't need quotation marks\
+export scout_multiplier=5
```
Comment (GrigorovskyA) [inline README.md:98]: Code uses other ENV variables. You can find it there - https://github.com/edeng23/binance-trade-bot/blob/master/binance_trade_bot/config.py

**Review by GrigorovskyA — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -45,64 +60,89 @@ Run the following line in the terminal: `pip install -r requirements.txt`.
...
--   Make sure that you have enough BNB in your account to pay the fee of the inspected trade.
--   Take into consideration the discount when calculating the trade threshold.
+
+You can [use BNB to pay for any fees on the Binance
+platform](https://www.binance.com/en/support/faq/115000583311-Using-BNB-to-Pay-for-Fees),
+which will reduce all fees by 25%. To support this benefit, the bot will always
+perform the following operations:
+
+1. Automatically detect that you have BNB fee payment enabled.
+1. Ensure that you have enough BNB in your account to pay the fee of the
```
Comment (Pekafu) [inline README.md:109]:
```suggestion
2. Ensure that you have enough BNB in your account to pay the fee of the
```

File: README.md [hunk 3]
```diff
@@ -45,64 +60,89 @@ Run the following line in the terminal: `pip install -r requirements.txt`.
...
+   inspected trade.
+1. Take into consideration the discount when calculating the trading threshold.
```
Comment (Pekafu) [inline README.md:111]:
```suggestion
3. Take into consideration the discount when calculating the trading threshold.
```

**Comment by edeng23** [comment]
Why are all of the lines marked as modified and not only the ones you've changed?

**Comment by edeng23** [comment]
I think that line wrapping should be enforced as a linting rule in order to avoid future inconsistencies.

File: README.md [hunk 4]
```diff
@@ -111,28 +151,42 @@ docker-compose up -d sqlitebrowser
...
 
 ## FAQ
 
-A list of answers to what seem to be the most frequently asked questions can be found in our discord server, in the corresponding channel.
+A list of answers to what seem to be the most frequently asked questions can be
+found in our discord server, in the corresponding channel.
 
 <p align="center">
-  <img src = "https://usercontent2.hubstatic.com/6061829.jpg">
+  <img src="https://usercontent2.hubstatic.com/6061829.jpg" width="500" height="500">
```
Comment (edeng23) [inline README.md:175]: The aspect ratio should be corrected, these dimensions stretch the image.

**Review by edeng23 — CHANGES_REQUESTED** [review]

**Comment by edeng23** [comment]
I'll close this PR now and make some changes to the README. I really appreciate the work you put in, and am only taking over because I think that it's important to update the outdated information as soon as possible.

**Review by 77mena — COMMENTED** [review]
Hello...im really sorry wont happen agine

**Pull request closed without merging**